pkgs/applications/terminal-emulators/mlterm/default.nix

```
@@ -19,25 +90,42 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-gfs5cdwUUwSBWwJJSaxrQGWJvLkI27RMlk5QvDALEDg=";
   };
 
-  nativeBuildInputs = [ pkg-config autoconf wrapGAppsHook ];
+  nativeBuildInputs = [
+    pkg-config
+    autoconf
+  ] ++ lib.optionals (enableTools.mlconfig) [
+    wrapGAppsHook
+  ];
   buildInputs = [
-    libX11
+    gtk
+    vte
+  ] ++ lib.optionals (imagelibChoice == "gdk-pixbuf") [
     gdk-pixbuf.dev
-    cairo
+  ] ++ lib.optionals (enableTypeEngines.xcore) [
```

Is this intentional or a leftover of previous function applications?

```suggestion
  ] ++ lib.optionals enableTypeEngines.xcore [
```

---

Leftover..